test/screenshot/material-icon/index.scss

```diff
@@ -1,3 +1,3 @@
-body {
-  margin: 24px;
+:global {
```

Comment:
If anything is imported from the packages directory, wrap it in `:global` so it isn't obfuscated.